src/agents/implementations/hebo-agent.ts

```
@@ -137,66 +160,60 @@ export class HeboAgent extends BaseAgent {
+        const output = response.output[0];
+        if (output.type === 'message' && output.status === 'completed') {
+          const content = output.content[0];
+          if (content.type === 'output_text') {
+            finalResponse = content.text;
+          }
+        }
+      }
+      // Extract the response content from the choices array (for Hebo API format)
+      else if (response.choices && response.choices.length > 0) {
+        const choice = response.choices[0];
+        if (choice.message && choice.message.content) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)